.github/workflows/deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+mkdir -p ~/.ssh
+ssh-keyscan -H www.skauting.cz >> ~/.ssh/known_hosts
```

Comment:
Funkční, ale bezpečnostně nic moc. Mnohem lepší by bylo to mít uložené v repu.

Reply:
Jako přímo si persistovat fingerprint? S tím souhlasím, tohle je 1:1 reimplementace Werckeru, ale můžeme to zároveň potunit.

https://github.com/skaut/Skautske-hospodareni/blob/bc0004be07ea8ecd52d53c0c33bcc5a9beb4f805/wercker.yml#L107-L110

Reply:
Předělal jsem to :+1: